chart/templates/reverse-proxy/deployment.yaml

```diff
@@ -26,15 +26,10 @@ spec:
         checksum/config: {{ include (print $.Template.BasePath "/reverse-proxy/configMap.yaml") . | sha256sum }}
     spec:
       {{- include "image.imagePullSecrets" . | nindent 6 }}
-      initContainers: 
```

Comment:
In https://github.com/huggingface/datasets-server/pull/2040 I forgot to remove this line, maybe that was the problem when deploying reverse-proxy? I didn't have any errors when running helm lint though.

Reply:
I'm not a YAML expert, it seems like an empty sequence should have `[]` indeed, so:

```
initContainers: []
```

would have been valid, while

```
initContainers: 
```

seems to be an invalid syntax.

https://stackoverflow.com/questions/5110313/how-do-i-create-an-empty-array-in-yaml

Reply:
I will try with initContainers: []

Reply:
hmmm, no, no need to add it, if it's empty we can just avoid having it in the code. I just wanted to say that `initContainers: ` seems to be an invalid syntax